ks3-jdk/src/jvmMain/kotlin/io/ks3/java/io/FileSerializer.kt

```diff
@@ -0,0 +1,7 @@
+package io.ks3.java.io
+
+import io.ks3.standard.stringSerializer
+import kotlinx.serialization.KSerializer
+import java.io.File
+
+object FileSerializer : KSerializer<File> by stringSerializer(::File, File::getPath)
```

Comment:
What do you think about serialising the file path in a system-independent way, using [`invariantSeparatorsPath`](https://kotlinlang.org/api/latest/jvm/stdlib/kotlin.io/java.io.-file/invariant-separators-path.html)? 

I think it would help to have stable output across systems, and since serialising is often used to communicate between different machines consistency would be best. UNIX separators are the de facto standard, and even JVM on Windows understands UNIX separated paths, even if it doesn't produce them by default.

Reply:
> What do you think about serialising the file path in a system-independent way, using [`invariantSeparatorsPath`](https://kotlinlang.org/api/latest/jvm/stdlib/kotlin.io/java.io.-file/invariant-separators-path.html)?

Good point, will do.